lib/TInitialState.php

```diff
@@ -171,6 +171,11 @@ protected function publishInitialStateForUser(IUser $user, IRootFolder $rootFold
 			'enable_matterbridge',
 			$this->serverConfig->getAppValue('spreed', 'enable_matterbridge', '0') === '1'
 		);
+
+		$this->initialState->provideInitialState(
```

Comment:
should not continue this and instead use capabilities for desktop client compatibility